deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/TwinConnectionState.java

```diff
@@ -10,9 +10,9 @@
  */
 public enum TwinConnectionState
 {
-    @SerializedName("disconnected")
+    @SerializedName("Disconnected")
```

Comment:
could we make this case-insensitive?

Reply:
I can't, but I can make it catch "disconnected" and "Disconnected"